core/sourcehook/sourcehook.h

```diff
@@ -196,11 +196,34 @@ namespace SourceHook
 			PassFlag_RetMem		= (1<<6),		/**< Object is returned in memory (through hidden first param */
 			PassFlag_RetReg		= (1<<7)		/**< Object is returned in EAX(:EDX) */
 		};
+		
+		enum RegisterType

```

Comment:
This is wrong, this file is dynamically generated. This needs to be in the hxx file.

Reply:
Oh thanks for your help, i added changes there and remove other commits